.storybook/stories/Colors.stories.mdx

```diff
@@ -1,8 +1,8 @@
-import { Meta, ColorPalette, ColorItem } from "@storybook/addon-docs";
+import { Meta, ColorPalette, ColorItem } from "@storybook/blocks";
 import brandTokens from "../../proprietary/Proprietary/dist/index.json";
 import { path2css } from "./util.js";
-import { MDXEmbedProvider } from "mdx-embed";
-const brandColorTokens = brandTokens["denhaag"]["color"];
+
+export const brandColorTokens = brandTokens["denhaag"]["color"];
```

Comment:
Is this export needed?

Reply:
Yes, it is mdx weirdness, you can only define a `const` if you export it, otherwise it will become regular text